pkg/topolvm-controller/cmd/run.go

```diff
@@ -130,6 +133,13 @@ func subMain() error {
 		return err
 	}
 
+	if err := mgr.AddHealthzCheck("healthz", healthz.Ping); err != nil {
+		return err
+	}
+	if err := mgr.AddReadyzCheck("readyz", healthz.Ping); err != nil {
```

Comment:
In this PR, two times of `AddReadyzCheck` function call are added. 
Could you tell me why it is needed to call `AddReadyzCheck` for both "webhook" and "readyz"?

Reply:
https://github.com/kubernetes-sigs/controller-runtime/blob/6c7c1d73fc5ed965c2962d6f4552b15f9841ce86/pkg/manager/internal.go#L253-L267
AddReadyzCheck accepts multiple checker instances. It may combine these results and returns a combined result from the http endpoint.

Reply:
When creating a new project from a current kubebuilder version, AddHealthzCheck/AddReadyzCheck is called using healthz.Ping like follow, so I set it:
https://github.com/kubernetes-sigs/kubebuilder/blob/v3.3.0/testdata/project-v3/main.go#L123-L130